.github/workflows/release-periodically.yml

```diff
@@ -24,9 +24,24 @@ permissions:
   contents: write
 
 jobs:
+  check-date:
```

Comment:
What are the benefits of having this done in a separate job? Couldn't we do it in the steps?